Source/Core/Core/HW/WiimoteEmu/WiimoteEmu.cpp

```diff
@@ -755,14 +780,43 @@ Common::Vec3 Wiimote::GetAcceleration()
   return accel;
 }
 
+Common::Vec3 Wiimote::GetAngularVelocity()
+{
+  // TODO: make cursor movement produce angular velocity.
+
+  auto orientation = Common::Matrix33::Identity();
+
+  // TODO: make a function out of this:
+  if (IsSideways())
+    orientation *= Common::Matrix33::RotateZ(float(MathUtil::TAU / -4));
+  if (IsUpright())
+    orientation *= Common::Matrix33::RotateX(float(MathUtil::TAU / 4));
+
+  return orientation * (m_tilt_state.angular_velocity + m_swing_state.angular_velocity);
+}
+
 Common::Matrix44 Wiimote::GetTransformation() const
 {
   // Includes positional and rotational effects of:
   // IR, Swing, Tilt
 
-  return Common::Matrix44::FromMatrix33(GetRotationalMatrix(-m_tilt_state.angle) *
-                                        GetRotationalMatrix(-m_swing_state.angle)) *
-         EmulateCursorMovement(m_ir) * Common::Matrix44::Translate(-m_swing_state.position);
+  // INFO_LOG(WIIMOTE, "ang.x: %f vel.x: %f", m_cursor_state.angle.x,
+  //        m_cursor_state.angular_velocity.x);
+
+  const auto rot_mtx = Common::Matrix44::FromMatrix33(GetRotationalMatrix(
+      //  -m_tilt_state.angle - m_swing_state.angle
+      -m_cursor_state.angle));
+
+  std::string text;
+  for (float f : rot_mtx.data)
+    text += " " + std::to_string(f);
```

Comment:
This should probably be removed, since it is only there for the `INFO_LOG` below.